core/src/services/hdfs/backend.rs

```diff
@@ -244,21 +244,21 @@ impl Access for HdfsBackend {
     type BlockingDeleter = oio::OneShotDeleter<HdfsDeleter>;
 
     fn info(&self) -> Arc<AccessorInfo> {
-        self.info.clone()
+        self.core.info()
     }
 
     async fn create_dir(&self, path: &str, _: OpCreateDir) -> Result<RpCreateDir> {
         let p = build_rooted_abs_path(&self.root, path);
 
-        self.client.create_dir(&p).map_err(new_std_io_error)?;
+        self.core.client.create_dir(&p).map_err(new_std_io_error)?;
 
         Ok(RpCreateDir::default())
     }
 
     async fn stat(&self, path: &str, _: OpStat) -> Result<RpStat> {
         let p = build_rooted_abs_path(&self.root, path);
```

Comment:
From this line to 272: `m.set_last_modified(meta.modified().into());` This can be a convenience function. Maybe as `HdfsCore::hdfs_get_metadata`. Then we can use `hdfs_get_metadata` in these functions:

- `blocking_stat`
- `stat`

`self.core.client.metadata` is a good function. We will keep using `self.core.client.metadata`.